astro/src/content/quickstarts/quickstart-springboot-web.mdx

```diff
@@ -56,188 +38,260 @@ The login flow will look like this after FusionAuth is introduced.
 
-The root of this project directory (next to this README) are two files, a Docker compose file and an environment variables configuration file. Assuming you have Docker installed on your machine, you can stand up FusionAuth up on your machine with:
+Start with getting FusionAuth up and running and creating a Spring Boot application with Thymeleaf.
+
+### Clone The Code
+
+First, grab the code from the repository and change into that directory.
 
 ```
 git clone https://github.com/FusionAuth/fusionauth-quickstart-springboot-web.git
 cd fusionauth-quickstart-springboot-web
+```
+
+### Run FusionAuth Via Docker
+
+You'll find a Docker Compose file `docker-compose.yml` and an environment variables configuration file `.env` in the root directory of the repo.
+
+Assuming you have Docker installed, you can start FusionAuth on your machine with the following.
+
+```
 docker-compose up -d
 ```
 
-The FusionAuth configuration files also make use of a unique feature of FusionAuth, called [Kickstart](/docs/v1/tech/installation-guide/kickstart): when FusionAuth comes up for the first time, it will look at the Kickstart file located at `kickstart/kickstart.json` and mimic API calls to configure FusionAuth for use when it is first run.
+This will start three containers, one each for FusionAuth, Postgres, and Elasticsearch.
 
-<Aside type="note" >If you ever want to reset the FusionAuth system, delete the volumes created by docker-compose by executing `docker-compose down -v`.</Aside>
+Here you are using a bootstrapping feature of FusionAuth, called [Kickstart](/docs/v1/tech/installation-guide/kickstart). When FusionAuth starts for the first time, it will look at the `kickstart/kickstart.json` file and configure FusionAuth to your specified state.
 
-FusionAuth will be initially configured with these settings:
+<Aside type="note" >If you ever want to reset the FusionAuth system, delete the volumes created by `docker-compose` by executing `docker-compose down -v`, then rerun `docker-compose up -d`.</Aside>
 
-* Your client Id is: `e9fdb985-9173-4e01-9d73-ac2d60d1dc8e`
-* Your client secret is: `super-secret-secret-that-should-be-regenerated-for-production`
+FusionAuth will be configured with these settings:
+
+* Your client Id is `e9fdb985-9173-4e01-9d73-ac2d60d1dc8e`.
+* Your client secret is `super-secret-secret-that-should-be-regenerated-for-production`.
 * Your example username is `richard@example.com` and your password is `password`.
 * Your admin username is `admin@example.com` and your password is `password`.
-* Your fusionAuthBaseUrl is 'http://localhost:9011/'
+* The base URL of FusionAuth is `http://localhost:9011/`.
 
-You can log into the [FusionAuth admin UI](http://localhost:9011/admin) and look around if you want, but with Docker/Kickstart you don't need to.
+You can log in to the [FusionAuth admin UI](http://localhost:9011/admin) and look around if you want to, but with Docker and Kickstart, everything will already be configured correctly.
```

Comment:
When I try to login to the admin UI I do not see the lock to log in.
![Screenshot 2023-09-19 at 8 25 02 PM](https://github.com/FusionAuth/fusionauth-site/assets/137117976/c91d2609-dd2b-4e9f-a50e-be8758e42b5b)

Reply:
Fixed in the PR here https://github.com/FusionAuth/fusionauth-quickstart-springboot-web/pull/1